lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+namespace :db do
```

Comment:
Sorry this has been waiting for so long, it's partially because I haven't heard of Nanobox before this PR. Regarding this task.. Is it necessary to check the db existence "before_live"? Could it not be something that runs only once when the nanobox is being created?